### PR TITLE
Updating sentences in jf99.txt

### PR DESCRIPTION
**Pull request by nmstoker**
Fix some spelling mistakes and a few awkward phrases

### CODE REVIEW

File: server/data/en/jf99.txt [hunk 1]
```diff
@@ -169,23 +169,23 @@ Taking part in road traffic is potentially dangerous after donating blood.
...
 This picture is pure art.
 A crowd of people stood around the fountain.
 When she entered the dance floor, everyone in the nightclub was looking at her.
 Wikipedia is a non-profit organization.
-The clock of the town center took damage from a lightening strike.
+The clock in the town center was damaged by a lightning strike.
 Water boarding is a terrible form of torture.
 The lawyer successfully defended his client.
-When the judge spoke the death sentence, the defendant showed no emotion.
+When the judge pronouced the death sentence, the defendant showed no emotion.
```
Comment (davidak) [inline server/data/en/jf99.txt]: You probably mean:
```suggestion
When the judge pronounced the death sentence, the defendant showed no emotion.
```

Comment (nmstoker) [reply server/data/en/jf99.txt]: Yes, you're quite right. Thanks for spotting that David!

**Comment by davidak** [comment]
Accepting the suggested changes created a new commit.

@Gregoor to have just one commit, squash and merge.

**Comment by Gregoor** [comment]
@davidak That's what I did. Squash and merge means that all commits will be rebased into a single commit (which is called squashing) and then merged. It will still a yield a commit, since that's the atomic unit of change in git.

**Pull request merged**